test/test_release.py

```diff
@@ -93,7 +93,7 @@ def md_changelog():
 
 ### Other changes
 
-- Changed `bitesize` label to `good first issue`. Our [contribute](https://github.com/beetbox/beets/contribute) page is now automatically populated with these issues. :bug: (\#4855)
+- Changed `bitesize` label to `good first issue`. Our [contribute](https://github.com/beetbox/beets/contribute) page is now automatically populated with these issues. :bug: (#4855)
```

Comment:
I don't understand why the backslashes were required in the first place and I guess you know what you are doing :-)

Reply:
That's just how Sphinx used to convert `rst` to `md` vs now: the backslashes are gone 😅